src/components/custom-field-input-currency/custom-field-input-currency.jsx

```diff
@@ -27,35 +27,36 @@ function getLocale() {
   return 'en-IN';
 }
 
-function initialInputValid(inputValue, currencyCode) {
-  const maximumFractionDigits = currencyMetaData[currencyCode].maximumFractionDigits;
-
+function initialInputValid(inputValue) {
   if (!inputValue) {
     return true;
   }
 
-  const splitInputValue = inputValue.toString().split('.');
-  if (splitInputValue.length === 1) {
+  if (inputValue.toString().split('.').length === 1) {
```

Comment:
This only splits on the American cents delineator `.` but do we need to worry about any others? I believe the pound uses `,` to separate parts from whole and the euro does as well.

Reply:
We don't care! Since we are expecting `value` to be cents, we never get a float. This is just here to ensure we actually don't pass it a float.